client/src/tests/RatingsAndReviews/ReviewsHeader.test.js

```diff
@@ -0,0 +1,65 @@
+import 'jsdom-global/register';
+
+// REACT
+import React from 'react';
+
+// TESTING
+import { shallow, mount } from 'enzyme';
+import { expect, jest, test, describe, beforeEach, afterEach } from '@jest/globals';
+import renderer from 'react-test-renderer';
+
+// COMPONENTS & HELPERS
+import ReviewsHeader from '../../RatingsAndReviews/Reviews/Reviews.jsx';
+import reviewRoutes from '../../clientRoutes/reviews.js';
+import helper from '../../helper-functions/rpHelpers.js';
+jest.mock('../../clientRoutes/reviews.js');
+const mockEvent = (attr, attrVal, mainValue) => {
```

Comment:
looks like your learning the way of the mock 🔥.  I just realized - we didnt even have to create a separate mocks folder.  i think we can just mockImplentation(()=>()) at top of file with whatever mocks you need and it will work for tests.  I need to change some of my mocks to return different values - so i changed my routes in qa mocks to just be jest.fn() - then in each before statement, i mock the data i want for the describe statement